addons/website/static/src/xml/website_form_editor.xml

```diff
@@ -332,7 +332,7 @@
                     class="s_website_form_input form-check-input"
                     t-att-id="recordId"
                     t-att-name="field.name"
-                    t-att-checked="record.selected and 'checked' or None"
+                    t-att="{checked: record.selected and 'checked' or None}"
```

Comment:
Just FYI, this has nothing to do with the change, but this template is evaluated in JS and `None` doesn't mean anything. It happens to work because it looks it up in the rendering context and gets back `undefined`, but it's a little confusing.